Formula/ghostunnel.rb

```diff
@@ -0,0 +1,20 @@
+class Ghostunnel < Formula
+  desc "Simple SSL/TLS proxy with mutual authentication"
+  homepage "https://github.com/ghostunnel/ghostunnel"
+  url "https://github.com/ghostunnel/ghostunnel/archive/refs/tags/v1.5.3.tar.gz"
+  sha256 "008b96d507b80480f4e78d31bdfbb37b794f51c2a7a80a8f3561844e2739b5a5"
+  license "Apache-2.0"
+  head "https://github.com/ghostunnel/ghostunnel.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "ghostunnel"
+    bin.install "ghostunnel"
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-X main.version=#{version}")
```
Makes it more transparent and in line with other `go`-based formulae.

Reply:
Done, rebased.